_includes/nav.html

```diff
@@ -1,46 +1,153 @@
+                  </ul>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </li>
+  </ul>
 </li>
-<li>
-  <a class="" href="//jakarta.ee/membership/">
-  Membership
+
+<li class="dropdown">
+  <a href="#" data-toggle="dropdown" class="dropdown-toggle">
+    <span>Specifications</span>
+    <b class="caret"></b>
   </a>
+  <ul class="dropdown-menu">
+    <li><a href="/specifications/">Browse Specifications</a> </li>
+    <li><a href="/about/jesp/">Specification Process</a> </li>
+    <li><a href="/committees/specification/guide/">JESP guide</a> </li>
+    <li><a href="https://projects.eclipse.org/list-of-projects?combine=jakarta&amp;field_project_techology_types_tid=1102&amp;field_state_value_2=All">Specification Projects</a> </li>
+  </ul>
 </li>
-<li>
-  <a class="" href="//jakarta.ee/connect/">
-  Connect
+
+<li class="dropdown">
+  <a href="#" data-toggle="dropdown" class="dropdown-toggle">
+    <span>Compatible Products</span>
+    <b class="caret"></b>
   </a>
+  <ul class="dropdown-menu">
+    <li><a href="/compatibility/">Jakarta EE Compatible Products</a> </li>
+    <li><a href="/compatibility/get-listed/">Get Listed</a> </li>
+  </ul>
 </li>
+
 <li class="dropdown">
   <a href="#" data-toggle="dropdown" class="dropdown-toggle">
-  <span>Resources</span>
-  <b class="caret"></b>
+    <span>Community</span>
+    <b class="caret"></b>
   </a>
   <ul class="dropdown-menu">
-    <li><a href="//jakarta.ee/resources#surveys">Surveys</a> </li>
-    <li><a href="//jakarta.ee/resources#infographics">Infographics</a> </li>
-    <li><a href="//jakarta.ee/resources#datasheets">Datasheets</a> </li>
-    <li><a href="//jakarta.ee/resources#presentations">Presentations</a> </li>
-    <li><a href="//jakarta.ee/community/adopt-a-spec/">Adopt A Spec</a> </li>
-    <li><a href="//jakarta.ee/committees">Committees</a> </li>
-    <li><a href="//jakarta.ee/newsletter/">Newsletter Archives</a> </li>
-    <li><a href="//jakarta.ee/meeting_minutes/">Meeting Minutes</a> </li>
-    <li><a href="https://Jakartablogs.ee">Blog</a> </li>
-    <li><a href="//jakarta.ee/resources#meetups">Tech Talk Meet-ups</a> </li>
-    <li><a href="https://www.youtube.com/channel/UC4M7h5b6elu9VlzjLfzuXyg">Jakarta EE YouTube Channel</a> </li>
-    <li><a href="https://projects.eclipse.org/list-of-projects?combine=jakarta&amp;field_project_techology_types_tid=1102&amp;field_state_value_2=All">Jakarta EE Specification Projects</a> </li>
-    <li><a href="https://drive.google.com/open?id=1kJFh6EIpOyAWP3JkymYZXOK311WM2HXm">Jakarta EE Community Folder</a> </li>
-    <li><a href="//jakarta.ee/resources#calendar">Jakarta EE Community Calendar</a> </li>
-    <li><a href="https://start.jakarta.ee/">Starter for Jakarta EE</a> </li>
-    <li><a href="https://jakartaone.org/">JakartaOne Livestream</a> </li>
+    <li><a href="/community/">Overview</a> </li>
```

Comment:
Links like these needs to be absolute since this theme will be used on a bunch of different domains.